uefi-test-runner/src/proto/media.rs

```diff
@@ -343,6 +344,38 @@ fn test_raw_disk_io2(handle: Handle) {
     }
 }
 
+fn test_disk_info() {
+    let disk_handles = uefi::boot::find_handles::<DiskInfo>().unwrap();
+
+    let mut found_drive = false;
+    for handle in disk_handles {
+        let disk_info = uefi::boot::open_protocol_exclusive::<DiskInfo>(handle).unwrap();
+        info!(
+            "DiskInfo at: {:?} (interface= {:?})",
+            handle,
+            disk_info.interface()
+        );
+        // Find our disk
+        if disk_info.interface() != DiskInfoInterface::SCSI {
+            continue;
+        }
+        let mut inquiry_bfr = [0; 128];
+        let Ok(len) = disk_info.inquiry(&mut inquiry_bfr) else {
+            continue;
+        };
+        if len >= 32 {
```

Comment:
isn't an assert better here? Otherwise, I see potential for introducing undetected errors

Reply:
The SCSI spec says a valid SCSI Inquiry response should be at least 36 bytes, so we can make an assert out of it I guess.